firmware/quadruna/CRIT/src/cubemx/Inc/FreeRTOSConfig.h

```diff
@@ -175,6 +175,7 @@ standard names. */
 
 /* USER CODE BEGIN Defines */
 /* Section where parameter definitions can be added (for instance, to override default ones in FreeRTOS.h) */
+#include "SEGGER_SYSVIEW_FreeRTOS.h"
```

Comment:
did you make this change? or was it autogen?

Reply:
This was autogen I believe.

Reply:
This is not autogen - it's a necessary change to get segger systemview working (our RTOS profiling tool)